src/couch_views/src/couch_views_util.erl

```diff
@@ -56,6 +57,12 @@ ddoc_to_mrst(DbName, #doc{id=Id, body={Fields}}) ->
                 DictBySrcAcc
         end;
         ({Name, Else}, DictBySrcAcc) ->
+            ?LOG_ERROR(#{
+                what => invalid_view_definition,
+                db => DbName,
+                ddoc => Id,
+                view => Name
```

Comment:
Noticed this doesn't include the `Else` bit. Probably for the better to avoid leaking internal design doc bits into the logs  (even when they are invalid).

Reply:
Yes, this was an intentional omission on my part. I didn't make many changes like this but I did try to avoid logging user data except for DB names and document IDs.